### PR TITLE
10.0 mig mgmtsystem hazard risk

### PR DESCRIPTION
**Pull request by ngrandjean**
Missing dependency : 
mgmtsystem_hazard

#159

### CODE REVIEW

File: mgmtsystem_hazard/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+=====
+
+To use this module, you need to:
+
+* go to Management Systems > Manuals > Hazard
+* create a new hazard to enter analysis information, risk evaluation, control measures, implementation tests and residual risks evaluation.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/128/9.0
```
Comment (max3903) [inline mgmtsystem_hazard/README.rst]: /10.0

File: mgmtsystem_hazard/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+management-system/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+management-system/issues/new?body=module:%20
+mgmtsystem_hazard%0Aversion:%20
+9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (max3903) [inline mgmtsystem_hazard/README.rst]: 10.0

**Review by max3903 — CHANGES_REQUESTED** [review]

**Comment by ngrandjean** [comment]
Checks of coverage failed but how can I avoid that ? I just changed what was necessary for the migration. Could it come from the fact that I migrated directly from 8.0 to 10.0 ?

**Comment by max3903** [comment]
@ngrandjean You would need to add new tests that execute the lines of code that are not currently covered.

**Review by max3903 — APPROVED** [review]

**Comment by ngrandjean** [comment]
I tried to add some tests but it didn't change anything for the coverage check (so I reverted that commit). What puzzles me is that the code didn't really change since the first tests where added.
Can someone help me with that ?
Thanks

**Pull request merged**